Python/Spotify-Downloader/README.md

```diff
@@ -0,0 +1,10 @@
+# Spotify Downloader
+Downloads songs, albums or playlists from Spotify.
+
+## Executing script
+  * Install dependencies using `pip install requirements.txt`
```

Comment:
requirements.txt is missing in you PR

Reply:
@tejan-singh Sorry, my bad. Added requirements.txt